apps/docs/content/components/UnorderedList/index.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Unordered list
+description: Use the unordered list component to render a list of unordered items.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing an unordered list with different bullet points.](https://github.com/primer/brand/assets/912236/646a4719-dddf-47f5-9b34-85f6addb828e)
+
+## Usage
+
+Use an unordered list to group related items together. It is suitable for presenting a collection of items that do not require a specific order or ranking. The unordered list provides visual bullet points for each item, which helps to emphasize the individuality of each point and makes it easier for users to scan and comprehend the information.
+
+Use the unordered list to effectively provide a quick reference or highlight key points. This allows users to easily grasp the main ideas or features without having to read lengthy paragraphs.
```

Comment:
This line feels more relevant to the next example. Could this be moved and replaced with guidance mentioning not to use it for content that requires linear grouping?